source/common/filesystem/kqueue/watcher_impl.cc

```diff
@@ -69,7 +69,7 @@ WatcherImpl::FileWatchPtr WatcherImpl::addWatch(const std::string& path, uint32_
   watch->callback_ = cb;
   watch->watching_dir_ = watching_dir;
 
-  int flags = NOTE_DELETE | NOTE_RENAME;
+  int flags = NOTE_DELETE | NOTE_RENAME | NOTE_WRITE;
```

Comment:
uint32_t ?

Reply:
It looks like `u_int` is the exact type used by kqueue. I'll switch to that.